packages/react/src/common/options.ts

```diff
@@ -64,14 +64,19 @@ export const DEFAULT_OPTIONS: ReactFlagEvaluationOptions = {
  */
 export const normalizeOptions: (options?: ReactFlagEvaluationOptions) => NormalizedOptions = (options?: ReactFlagEvaluationOptions) => {
   const defaultOptionsIfMissing = !options ? {} : options;
```

Comment:
Just a nit, you can do `options ?? {}` here.

Reply:
absolutely. I hate when I see this. Thanks.

Reply:
I'm also seeing the var name is now misleading, I'll fix that too.

Reply:
Actually I can just default this at the param level!